examples/multiview.cpp

```diff
@@ -0,0 +1,116 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "RAJA/RAJA.hpp"
+#include <cstdio>
+#include <array>
+
+/*
+ * MultiView Usage Example
+ *
+ * RAJA::MultiView is a view on an array-of-pointers
+ * (or pointer-to-pointers), rather than a RAJA::View on a single
+ * array. This allows a single RAJA::Layout to be applied to a
+ * large set of arrays, and encapsulates a set of arrays more
+ * easily.
+ *
+ * Instantiation of a MultiView works exactly like a standard View:
+ *
+ * RAJA::MultiView<data_type, RAJA::Layout<dim,index_type>> myMView(arrayofpointers, layout)
+ *
+ * Accessing the MultiView is similar to View, with an additional
+ * array-of-pointers index in the 0th position. The following example
+ * obtains the (x,y)th element of the 3rd array in an array-of-pointers:
+ *
+ * myMView(3, x, y);
+ *
+ * Optionally, the position of the array-of-pointers index can be
+ * "swizzled" within the MultiView access. This is an example of the
+ * same previous access, with the array-of-pointers index swizzled
+ * to the 2nd position (the default position used in the previous 
+ * example is the 0th position):
+ *
+ * RAJA::MultiView<data_type, RAJA::Layout<dim,index_type>, 2> myMView(arrayofpointers, layout)
+ * myMView(x, y, 3);
+ *
+ * The following code demonstrates 2 aspects of RAJA::MultiView usage:
+ * - Basic usage
+ * - Swizzling of the array-of-pointers index
```

Comment:
Please, remove the term "swizzle" everywhere and use a more explicit explanation of what's actually going on.

Reply:
@rhornung67 Ok, I think I've addressed all of the documentation and example changes. The documentation has been reworked to start from a simpler 1-D access and finishes with a 2-D access, so it should be more comprehensible.